prac_05/emails.py

```diff
@@ -0,0 +1,15 @@
+email_name = {}
+email = input("Email: ")
+valid = False
+while email != "":
+    name = email.split("@")
+    name = name[0]
```

Comment:
Could this be made more efficient by combining into one line using email.split("@")[0]?